core/src/test/java/org/kohsuke/stapler/export/NamedPathPrunerTest.java

```diff
@@ -56,10 +56,10 @@ public void testRange() throws Exception {
             jobs[i] = new Jhob("job"+i,"aaa","bbb");
         Vhew v = new Vhew("view","aaa",jobs);
 
-        assertResult("{jobs:[{name:job0},{name:job1},{name:job2}]}",    v, "jobs[name]{,3}");
-        assertResult("{jobs:[{name:job3},{name:job4},{name:job5}]}",    v, "jobs[name]{3,6}");
-        assertResult("{jobs:[{name:job38}]}",                           v, "jobs[name]{38}");
-        assertResult("{jobs:[{name:job97},{name:job98},{name:job99}]}", v, "jobs[name]{97,}");
+        assertResult("{class:Vhew,jobs:[{class:Jhob,name:job0},{class:Jhob,name:job1},{class:Jhob,name:job2}]}",    v, "jobs[name]{,3}");
+        assertResult("{class:Vhew,jobs:[{class:Jhob,name:job3},{class:Jhob,name:job4},{class:Jhob,name:job5}]}",    v, "jobs[name]{3,6}");
+        assertResult("{class:Vhew,jobs:[{class:Jhob,name:job38}]}",                           v, "jobs[name]{38}");
+        assertResult("{class:Vhew,jobs:[{class:Jhob,name:job97},{class:Jhob,name:job98},{class:Jhob,name:job99}]}", v, "jobs[name]{97,}");
```

Comment:
It would be nicer if the meta-attribute could be omitted where it adds no information: where the `@Exported` attribute’s type (or element type) in fact matches the concrete type of the value. In this case for example, that would mean that `class:Jhob` would not be sent, since `Vhew.jobs` is not a heterogenerous list.

Whether the meta-attribute is desirable at top level is a little unclear. Usually it is superfluous, yet there are some cases where you want it. For example, to distinguish different top-level item types, the [XML output](https://jenkins.ci.cloudbees.com/job/core/job/jenkins-core/api/xml) currently gives a hint in the root element name (a mangled version of the class’ `simpleName`, I think), whereas the [JSON output](https://jenkins.ci.cloudbees.com/job/core/job/jenkins-core/api/json) gives no indication. So it is better to include it in this case. Anyway there is only one such attribute per HTTP request—not much effort.